velox/functions/sparksql/RegisterArithmetic.cpp

```diff
@@ -66,6 +67,27 @@ void registerArithmeticFunctions(const std::string& prefix) {
   registerFunction<HypotFunction, double, double, double>({prefix + "hypot"});
   registerFunction<sparksql::Log2Function, double, double>({prefix + "log2"});
   registerFunction<sparksql::Log10Function, double, double>({prefix + "log10"});
+
+  // Rand functions.
+  registerFunction<RandFunction, double>({prefix + "rand", prefix + "random"});
```

Comment:
Let's refactor to extract helper function to avoid making this function so large.

Reply:
Just refactored and rebased the code. Thanks!

Reply:
@PHILO-HE Thank you. I'll try to land this now.